src/app/loadout-drawer/subclass-drawer/ItemSocketsSubclass.tsx

```diff
@@ -120,24 +121,29 @@ function SocketForCategory({
   dimSocket: DimSocket;
   onClick(): void;
 }) {
+  const [itemSize] = useSetting('itemSize');
   if (
     socketCategory.category.categoryStyle === DestinySocketCategoryStyle.Supers &&
     dimSocket.plugged
   ) {
+    const viewBoxMax = itemSize - 1;
+    const viewBoxHalf = itemSize / 2 - 1;
     return (
       <PressTip tooltip={<DimPlugTooltip item={item} plug={dimSocket.plugged} />}>
-        <svg viewBox="0 0 49 49" className={styles.super}>
-          <image xlinkHref={bungieNetPath(dimSocket.plugged.plugDef.displayProperties.icon)} />
-          <polygon
-            strokeDasharray="265.87216"
-            style={{ strokeDashoffset: 0 }}
-            fillOpacity="0"
-            stroke="#ddd"
-            strokeWidth="1"
-            points="24,0 49,24 24,49 0,24"
-            strokeLinecap="butt"
-          />
-        </svg>
+        <div className={styles.super}>
+          <BungieImage src={dimSocket.plugged.plugDef.displayProperties.icon} />
+          <svg viewBox={`0 0 ${viewBoxMax} ${viewBoxMax}`} className={styles.superOutline}>
```

Comment:
I'm not sure about this. Given a correct (fixed size) view box, you should be able to scale the overall image using height/width - that's one of the main benefits of SVG.

Reply:
Yeah I can revert that, it was an attempt to figure out what might work.